app/components/job-tabs.js

```diff
@@ -11,5 +12,11 @@ export default Component.extend({
     if (isEmpty(this.$('.travistab-nav--secondary').find('.active'))) {
       this.$('#tab_log').addClass('active');
     }
-  }
+  },
+
+  tabTitle: computed('job.build.request.noYaml', function () {
+    if (this.get('job.build.request.noYaml')) { return null; }
```

Comment:
It seems like `job.build.request.noYaml` worth an alias, as it already has multiple occurrences in template and now in component too.
```js
noYaml: reads('job.build.request.noYaml')
```

In this case this computed property could be simplified as well:
```js
tabTitle: computed('noYaml', function () {
  return this.noYaml ? '' : 'View this job's config';
})
```